charts/kube-prometheus-stack/Chart.lock

```diff
@@ -10,3 +10,4 @@ dependencies:
   version: 6.37.3
 digest: sha256:1db99e2752cc271323cf886c3ea14105f3aec0883ec67e944533f6246870d15c
 generated: "2022-09-11T15:48:48.5400618+03:00"
+
```

Comment:
Looks like this blank line got added by accident.

Reply:
fixed